### PR TITLE
Feature/Placement

### PR DESCRIPTION
**Pull request by avcio9**

### CODE REVIEW

**Comment by QuentinGruber** [comment]
Shouldn't we merge this ? @Z1Meme @avcio9. Since it's already used on community server.

**Comment by avcio9** [comment]
there is an issue with with simpleNpcs constructions, their rotation gets tweaked sometimes, also at this point we cant have hp bars to show up using lightWeightNpcs, so for now we can either have hp bars on bases or proper rotation. 
Pr still needs a little more work,

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
```
Comment (QuentinGruber) [inline .vs/VSWorkspaceState.json]: we need to add .vs/ to .gitignore

**Review by Z1Meme — APPROVED** [review]
👍

**Pull request merged**